samples/README.md

```diff
@@ -0,0 +1,14 @@
+# Getting Started with Datastore Samples
+
+## Running a sample using Cloud Shell
+
+The Google Cloud Shell has application default credentials from its compute instance which will allow you to run samples without having to obtain `GOOGLE_APPLICATION_CREDENTIALS`.
+Go to the [Datastore Client Readme](https://github.com/googleapis/java-datastore#samples) to run each sample in the Cloud Shell.
+
+## Running a sample at the command line
+
+First set up `GOOGLE_APPLICATION_CREDENTIALS` and `GOOGLE_CLOUD_PROJECT` environment variables before running any samples.
```

Comment:
The shouldn't need to set `GOOGLE_APPLICATION_CREDENTIALS` if they use `gcloud auth application-default login`

Reply:
Fixed, thanks!